CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ Please document all notable changes to this project in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [v0.4.0]
```

Comment:
Until there is an actual release, this should be `[Unreleased]`.

Reply:
I'd like to cut a release once this is merged.